specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/blob.json

```diff
@@ -544,7 +544,7 @@
             "in": "header",
             "required": false,
             "type": "string",
-            "description": "The entity state (ETag) version of the immutability policy to update. A value of \"*\" can be used to apply the operation only if the immutability policy already exists. If omitted, this operation will always be applied."
+            "description": "The entity state (ETag) version of the immutability policy to update must be returned to the server for all update operations."
```

Comment:
Per the descritpion, is there a breaking change of the behavior on server (omit if-Match, before will apply the operation, now will fail?)
Besides that, if the format is still \"*\", please don't miss it in the updated description.

Reply:
Hi @blueww, thanks for the review. This PR is only a description change for correctness, and there is no change in server behavior.

The current description in our documentation is wrong. We have never supported "*" as a valid value on the server, so we are removing it.
Also, the if-match has always been required for Update operations. The "required" field is "false" for GET and "true" for PUT and other updates.

Reply:
@AbhishekKrishna123 

Thanks for the reply!

For `\"*\"`, I think it means there should be double quota in the begin/end of etag. 
If this is not the proper way to show that, it's better describe it in another way, but not remove the info from the description.

I am good with other part of the description change.

Reply:
@blueww We never supported "*" for the eTag value in our service, it is not related to the quotes. The documentation has been wrong since many years when the feature was introduced. Adding @HimanshuChhabra from my team for a review form our side.

Reply:
The description change looks good to me, SRP does not support * or "*" values for e-tag.

Reply:
@AbhishekKrishna123 

I believe the * here is a wild char, `\"*\"` is to show the etag need have begin/end double quota.
It's OK for me to remove `\"*\"` , however we should let customer know etag need begin/end double quota in the description, to avoid customer failure.

Reply:
> I just reconfirmed that the etag value that we are returning from the service includes double quotes and the double quotes is required for the request to succeed. For example: ""8dca4aced6027dd"",

Yes, so the description should still indicate the double quotes need be in included in the etag (as the description before this PR indicate that with `\"*\"`).

Reply:
Sure, I will update the PR

Reply:
@blueww Please check if this change looks good. Added "The ETag value must include the leading and trailing double quotes as returned by the service."

Reply:
Thanks for the update!
Looks good to me.
I have approved the PR.